apps/deepl_integration/serializers.py

```diff
@@ -122,9 +122,10 @@ class UnifiedConnectorLeadExtractCallbackSerializer(DeeplServerBaseCallbackSeria
         child=serializers.CharField(allow_blank=True),
         required=False, default=[],
     )
-    text_path = serializers.CharField(required=False)
-    total_words_count = serializers.IntegerField(required=False, default=0)
-    total_pages = serializers.IntegerField(required=False, default=0)
+    text_path = serializers.CharField(required=False, allow_null=True)
+    total_words_count = serializers.IntegerField(required=False, default=0, allow_null=True)
+    total_pages = serializers.IntegerField(required=False, default=0, allow_null=True)
+    text_extraction_id = serializers.CharField(required=False, allow_null=True)
```

Comment:
We need to add validation after adding allow_null=True to make sure required field are provided when the status is success.